stix2validator/test/indicator_tests.py

```diff
@@ -140,6 +140,31 @@ def test_invalid_pattern(self):
         indicator['pattern'] = """[file:hashes."SHA-256" = 'aec070645fe53ee3b3763059376134f058cc337247c978add178b6ccdfb0019f']"""
         self.assertFalseWithOptions(json.dumps(indicator))
 
+    def test_pattern_custom_invalid_format(self):
+        indicator = copy.deepcopy(self.valid_indicator)
```

Comment:
Yes, several tests modify the object; for instance several in this file modify the indicator's pattern. I suppose it wouldn't be any different if we just did json.loads() in each test.